src/test/java/steps/NewCheckingWithLowAmountSteps.java

```diff
@@ -0,0 +1,57 @@
+package steps;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import models.NewCheckingAccountInfo;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.safari.SafariDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import pages.LoginPage;
+import pages.NewCheckingPage;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class NewCheckingWithLowAmountSteps {
+    WebDriver driver = new SafariDriver();
+    private LoginPage loginPage = new LoginPage(driver);
+    private NewCheckingPage newCheckingPage = new NewCheckingPage(driver);
+
+    @Before
+    public void theUserIsOnTheWebpage() {
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+        driver.get("https://dbank-qa.wedevx.co/bank/login");
```

Comment:
What do you think about making this taken from the feature file like we used to do before?